quickstarts/maven/spring-boot-with-yaml/pom.xml

```diff
@@ -0,0 +1,169 @@
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>1.4.1.RELEASE</version>
+  </parent>
+
+  <groupId>org.eclipse.jkube.quickstarts.maven</groupId>
+  <artifactId>spring-boot-yaml</artifactId>
+  <version>0.2.1-SNAPSHOT</version>
+  <name>Eclipse JKube :: Quickstarts :: Maven :: Spring Boot - YAML</name>
+  <packaging>jar</packaging>
+
+  <description>Minimal Example with Spring Boot and YAML files</description>
+
+  <dependencies>
+
+    <!-- Boot generator  -->
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-actuator</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.jolokia</groupId>
+      <artifactId>jolokia-core</artifactId>
+    </dependency>
+
+  </dependencies>
+
+  <build>
+    <plugins>
+
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+      </plugin>
+
+
+    </plugins>
+  </build>
+
+  <profiles>
+    <profile>
+      <id>kubernetes</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.eclipse.jkube</groupId>
+            <artifactId>kubernetes-maven-plugin</artifactId>
+            <version>${project.version}</version>
+
+            <configuration>
+              <resources>
+                <labels>
+                  <all>
+                    <testProject>spring-boot-with-yaml-label-for-all</testProject>
+                  </all>
+                </labels>
+              </resources>
+              <generator>
+                <config>
+                  <spring-boot>
+                    <color>always</color>
+                  </spring-boot>
+                </config>
+              </generator>
+              <enricher>
+              </enricher>
+<!--              <helm>-->
+<!--                <chart>This is a Test</chart>-->
+<!--                <type>kubernetes</type>-->
+<!--                <icon>This is the icon</icon>-->
+<!--                <sources>-->
+<!--                  <source>https://different.source/yaml.yml</source>-->
+<!--                </sources>-->
+<!--                <maintainers>-->
+<!--                  <maintainer>-->
+<!--                    <name>John</name>-->
+<!--                    <email>john.doe@example.com</email>-->
+<!--                  </maintainer>-->
+<!--                </maintainers>-->
+<!--              </helm>-->
```

Comment:
Is it okay to leave it commented here like this? I was thinking if we could have a dedicated sample for helm. Maybe we can rename this sample to `spring-boot-helm`... WDYT?

Reply:
The name was inherited from FMP, I can rename to helm (it's clearer to me than "-yaml"

Regarding the commented-out section, this is a Quickstart. The purpose is to showcase how the plugin behaves given certain configurations, we can either create more granular quickstarts, add more profiles, or leave commented-out sections such as this one.

I opted for this approach because creating too many Quickstarts increase the maintenance complexity and also may pollute the code-base if there is no sort of criteria when defining the examples.
Using maven profiles is also a way to achieve this, but the pom will be more difficult to read and get messy.
Using commented out sections has the advantage that we can actually show what configurations are supported by the Mojo with examples and the user can un-comment each configuration to see how it behaves. This is not production code but sample code it should be verbose and documented.

The  `<enricher></enricher>` must be removed.